code/AssetLib/MDL/MDLMaterialLoader.cpp

```diff
@@ -493,12 +493,12 @@ void MDLImporter::ParseSkinLump_3DGS_MDL7(
 
         aiString szFile;
         const size_t iLen = strlen((const char *)szCurrent);
-        size_t iLen2 = iLen + 1;
-        iLen2 = iLen2 > MAXLEN ? MAXLEN : iLen2;
+        size_t iLen2 = iLen > MAXLEN - 1 ? MAXLEN - 1: iLen;
```

Comment:
You should add explicit parens here. I had to go look up C++ precedence rules to verify that this is correct. I think it is correct but people shouldn't have to do that.